README.md

```diff
@@ -24,104 +18,67 @@ of active locales is:
 
 ### Setting up to edit
 
-This repo has exactly the same folder structure, concepts, and commands
-available to it as the [content repo](https://github.com/mdn/content), which
-holds all of MDN's English content. The main difference is in the setup you need
-to do before you can start editing. It is mostly the same, but there is a little
-bit more to consider.
+This repo has exactly the same folder structure, concepts, and commands available to it as the [content repo](https://github.com/mdn/content), which holds all of MDN's English content. The main difference is in the setup you need to do before you can start editing. It is mostly the same, but there is a little bit more to consider.
 
-To begin with, get the basic required tooling set up, as described in the
-[content repo Setup section](https://github.com/mdn/content#setup).
+To begin with, get the basic required tooling set up, as described in the [content repo Setup section](https://github.com/mdn/content#setup).
 
-Now you need to fork and clone both the
- [content repo](https://github.com/mdn/content)
- and the translated-content repo (this repo).
+Now you need to fork and clone both the [content repo](https://github.com/mdn/content) and the translated-content repo (this repo).
 
 ### Content repo setup
 
 1. Once the above is done, cd into the content repo.
 
-2. Run the command `yarn install` to fetch the latest packages and get the local
-   MDN testing environment set up. It is also recommended that you run
-   `yarn install` before every update you do to the source, to make sure you
-   have the latest packages.
+2. Run the command `yarn install` to fetch the latest packages and get the local MDN testing environment set up. It is also recommended that you run `yarn install` before every update you do to the source, to make sure you have the latest packages.
 
-3. Next, create an environment variable called `CONTENT_TRANSLATED_ROOT`
-   containing the path to the *translated-content* repo’s `files` directory. You
-   could do this for a single session like so:
+3. Next, create an environment variable called `CONTENT_TRANSLATED_ROOT` containing the path to the *translated-content* repo’s `files` directory. You could do this for a single session like so:
 
    ```bash
    export CONTENT_TRANSLATED_ROOT=/path/to/translated-content/files
    ```
 
-   But you'll have to newly-set this every time you open up a new terminal
-   window. Instead, you could put the environment variable setting in an `.env`
-   file in the root of your content repo. This is most easily done using the
-   following command:
+   But you'll have to newly-set this every time you open up a new terminal window. Instead, you could put the environment variable setting in an `.env` file in the root of your content repo. This is most easily done using the following command:
```

Comment:
```suggestion
   You must set this every time you start a new shell or terminal session. To avoid this, you can put the environment variable in a `.env` file at the root of your content repo. You can do this with the following command:
```